profiles/mvebu-generic.profiles

```diff
@@ -0,0 +1,14 @@
+armada-370-db
+armada-370-rd
+armada-385-db-ap
+armada-388-clearfog-base
+armada-388-clearfog-pro
```

Comment:
From my former researches I remember the ClearFrog boards are only PCBs w/o chassis or PSU. 
Beside the Linksys boards all others seem to be some kind of Developmentboards and no "enduser products", so not sure if it makes sense to build these